kernel/build/build_rpm

```diff
@@ -84,6 +84,19 @@ if [ "$THROWAWAY" = false ] ; then
     # push binaries to chacra
     find ~/rpmbuild/ | egrep '\.rpm$' | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/$ARCH/
 
+    # write json file with build info
+    cat > $WORKSPACE/repo-extra.json << EOF
+{
+    "version":"$kernelrelease",
```

Comment:
@andrewschoen Currently we have builder.version property that corresponds to extra:package_manager_version. @zmc and I agreed to add another property, builder.scm_version, that would correspond to extra:version, to be used by the kernel task. builder.scm_version can either be ShamanProject-specific or just return builder.version in the GitbuilderProject case.

The rationale here is that the kernel task has always been special and kernel gitbuilder /version file never contained an actual package version.